scripts/dataverse/migrate_to_external_account.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Script to migrate dataverse projects to accomodate changes in their 4.0 API.
+This assumes that the script `migrate_credentials_to_token` has been run.
```

Comment:
Nope, you can run the script by itself. This is just me copying my old script and not reading anything above the import statements.